helm_deploy/values-dev.yaml

```diff
@@ -18,6 +18,11 @@ generic-service:
     ATHENA_OUTPUT_BUCKET: "s3://emds-test-athena-query-results-20240923095933297100000013"
     MFA_REQUIRED: "false"
 
+  namespace_secrets:
+    hmpps-electronic-monitoring-datastore-dev-athena-roles:
```

Comment:
as the config is now per environment should we remove the environment name from the namespace?